python/opendataloader-pdf-mcp/tests/test_convert_pdf.py

```
@@ -83,6 +91,103 @@ def test_quiet_is_always_true(self, input_pdf, tmp_path):
+    temp dir that's already been deleted.
+    """
+
+    def test_markdown_embeds_images_as_base64_by_default(self, input_pdf_with_images):
+        """First page of the academic sample has images; output must inline them."""
+        result = convert_pdf(
+            input_path=str(input_pdf_with_images), format="markdown", pages="1"
+        )
+        assert "![" in result, "expected markdown image syntax in output"
+        assert "data:image/" in result and ";base64," in result, (
+            "expected base64-embedded image data URI in markdown output"
+        )
```

Ruff (0.15.15)

[warning] Assertion should be broken down into multiple parts

(PT018)